src/Core/src/Platform/iOS/ViewExtensions.cs

```diff
@@ -504,7 +504,9 @@ internal static Size LayoutToMeasuredSize(this IView view, double width, double
 
 		public static void UpdateInputTransparent(this UIView platformView, IViewHandler handler, IView view)
 		{
-			if (view is ITextInput textInput)
+			// Interaction should not be disabled for an editor if it is set as read-only
+			// because this prevents users from scrolling the content inside an editor.
+			if (view is not IEditor && view is ITextInput textInput)
```

Comment:
I wonder if this is all wrong for IEntry too and we should be using the https://learn.microsoft.com/en-us/dotnet/api/uikit.uitextfielddelegate.shouldbeginediting?view=xamarin-ios-sdk-12

But this will still apply for the search bar. So we can do this (investigate at least) later.